app/views/charge.njk

```diff
@@ -349,7 +349,13 @@
                     {{ __p("commonConjunctions.or") }}
                   </span>
                   <img src="/images/amex-security-code.png" class="amex-cvc hidden" alt="{{ __p("cardDetails.amexcvcTip") }}"/>
-                </div>
+                 </div>
+                   {% if savePaymentInstrumentToAgreement and agreementId  and agreementDescription %}
+                   <div>
```

Comment:
The other blocks in this form use the design system `govuk-form-group` to help group together components and make sure there's enough space, I think I'd recommend that was done here too:
```html
<div class="govuk-form-group">
    <div class="govuk-inset-text">
        <p class="govuk-body">Your payment details will be saved for:</p>
        <p class="govuk-body">{{ agreementDescription }}</p>
    </div>              
</div>
```